docs/src/operate/c8y/supported-operations.md

```diff
@@ -241,9 +243,177 @@ Arguments will be parsed correctly as long as following features are not include
 
 In case those unsupported shell features are present, the syntax that introduce them is interpreted literally.
 
-Be aware that SmartREST payload is always added as the last argument. The command presented above will actually lead to following code execution
+Be aware that SmartRest payload is always added as the last argument. The command presented above will actually lead to following code execution
 
 ```bash
 python /etc/tedge/operations/command.py $SMART_REST_PAYLOAD
 ```
-:::
\ No newline at end of file
+:::
+
+#### Create operation delivered via C8Y Json
```

Comment:
```suggestion
#### Create operation delivered via Cumulocity JSON over MQTT
```

P.S. [It seems officially it is **JSON via MQTT**](https://cumulocity.com/docs/smartrest/json-via-mqtt/). However, we always call **JSON over MQTT**.

Reply:
Resolved by a267f04581ec93b1644543df4d45e998b590aaa8